packages/core/src/services/angle-utils.ts

```diff
@@ -80,16 +80,12 @@ export function polarToCartesianCoords(
 	r: number,
 	t: Point = { x: 0, y: 0 }
 ): Point {
-	const x = r * Math.cos(a) + t.x;
-	const y = r * Math.sin(a) + t.y;
+	let x = r * Math.cos(a) + t.x;
+	let y = r * Math.sin(a) + t.y;
```

Comment:
why would x & y not be `const`s?

Reply:
Missed that during the previous fix, I have just changed the variable declarations to `const`.